source/fem-dem/gls_sharp_navier_stokes.cc

```diff
@@ -1637,14 +1637,21 @@ GLSSharpNavierStokesSolver<dim>::integrate_particles()
           double d_residual_dv =
             -bdf_coefs[0] -
             0.5 * volume * fluid_density / particles[p].mass / dt + DBL_MIN;
```

Comment:
I think the name of this variable is not clear, as I couldn't understand exactly what this was. After talking to Lucka, I suggest to change the name to something related to alpha, as this is no longer the original definition of `d_residual_dv`. The same comment applies to all the following definitions of `d_residual_dv` and `d_residual_domega`

Reply:
All the variables name use in the calculation of the update have been updated to more verbose name